# can be "marker" or "room"
string type
---
bool success
